client/src/components/AreaDetail/tests/__snapshots__/areaDetail.test.tsx.snap

```diff
@@ -11,27 +11,31 @@ exports[`rendering of the AreaDetail checks if various text fields are visible 1
           Census block group:
         </span>
         <span>
-          98729374234
+           98729374234
         </span>
       </li>
       <li>
         <span>
           County:
         </span>
-        <span />
+        <span>
+           undefined
```

Comment:
The `undefined` here could be an indicator of a deeper issue.  (e.g. `foo!.bar` vs `foo!.bar || ''` or some variable/param needing a reasonable default value).